js/app/dashboard/index.js

```diff
@@ -130,6 +130,21 @@ function start() {
     }
   }
 
+  // the empty tour panels are initially open
+  let expandedToursIds = []
+  tours.forEach((tour) => {if (tour.itemIds.length == 0) {expandedToursIds.push(tour['@id'])}})
+  
+  preloadedState = {
+    ...preloadedState,
+    logistics: {
+      ...preloadedState.logistics,
+      ui: {
+        ...preloadedState.ui,
+        expandedTourPanelsIds: expandedToursIds
+      }
+    }
+  }
+
```

Comment:
I think it could be made a bit simpler using lodash merge https://lodash.com/docs/4.17.15#merge

```
_.merge(preloadedState, {
  logistics: {
    ui: {
      expandedTourPanelsIds: expandedToursIds
    }
  }
})
```